package.json

```diff
@@ -10,9 +10,8 @@
     "flow.copy:src": "./node_modules/.bin/cpy ng2-select.ts \"components/*.ts\" ts --parents",
     "flow.clean": "./node_modules/.bin/del bundles coverage demo-build typings \"components/**/*.+(js|d.ts|js.map)\" dist \"ng2-select.+(js|d.ts|js.map)\"",
     "flow.deploy:gh-pages": "npm run flow.build:prod && ./node_modules/.bin/gh-pages -d demo-build",
-    "flow.eslint": "./node_modules/.bin/eslint --ignore-path .gitignore  --ext js --fix . .config",
-    "flow.tslint": "./node_modules/.bin/gulp lint",
-    "flow.lint": "npm run flow.eslint && npm run flow.tslint",
+    "flow.eslint": "./node_modules/.bin/eslint --ignore-path .eslintignore  --ext js --fix . .config",
+    "flow.lint": "npm run flow.eslint",
```

Comment:
May be you are right, let me check

On Fri, Apr 22, 2016, 18:18 Jonathon Lee <notifications@github.com> wrote:

> In package.json
> <https://github.com/valor-software/ng2-select/pull/153#discussion_r60755694>
> :
>
> > @@ -10,9 +10,8 @@
> >      "flow.copy:src": "./node_modules/.bin/cpy ng2-select.ts \"components/*.ts\" ts --parents",
> >      "flow.clean": "./node_modules/.bin/del bundles coverage demo-build typings \"components/**/*.+(js|d.ts|js.map)\" dist \"ng2-select.+(js|d.ts|js.map)\"",
> >      "flow.deploy:gh-pages": "npm run flow.build:prod && ./node_modules/.bin/gh-pages -d demo-build",
> > -    "flow.eslint": "./node_modules/.bin/eslint --ignore-path .gitignore  --ext js --fix . .config",
> > -    "flow.tslint": "./node_modules/.bin/gulp lint",
> > -    "flow.lint": "npm run flow.eslint && npm run flow.tslint",
> > +    "flow.eslint": "./node_modules/.bin/eslint --ignore-path .eslintignore  --ext js --fix . .config",
> > +    "flow.lint": "npm run flow.eslint",
>
> There are 2 changes here.
>
>    - .gitignore and .eslintignore are fundamentally different things
>    - remove use of gulp ts lint in favour of webpack ts linting,
>
> I actually think it would be easier to use just webpack for almost
> everything, but maybe that's just personal preference
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/valor-software/ng2-select/pull/153/files/bba35f459e80f9533e8a1ad29f6010540b499fd7#r60755694>
>